### PR TITLE
Update u2f failure dialog to show support

### PR DESCRIPTION
**Pull request by mitsuhiko**
screen shot 2016-08-30 at 19 19 37

@getsentry/ui
@getsentry/infrastructure 

---

This change is [Reviewable](https://reviewable.io/reviews/getsentry/sentry/4043)

### CODE REVIEW

File: src/sentry/options/defaults.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 # System
 register('system.admin-email', flags=FLAG_REQUIRED)
+register('system.support-email', flags=FLAG_ALLOW_EMPTY)
```
Comment (mattrobenolt) [inline src/sentry/options/defaults.py]: Add `FLAG_PRIORITIZE_DISK` flag.

**Comment by dcramer** [comment]
This is a step forward, but we need to do a UI overhaul on the authentication pages (as a future followup)

**Comment by mitsuhiko** [comment]
Review status: 0 of 4 files reviewed at latest revision, 1 unresolved discussion.

---

*[src/sentry/options/defaults.py, line 23 \[r1\]](https://reviewable.io:443/reviews/getsentry/sentry/4043#-KQRScuOt9MCzNWZLHbU-r1-23:-KQRTzZSeCw9b1PSbAe7:b-896fix) ([raw file](https://github.com/getsentry/sentry/blob/fe65cff9208a1ae268cdf62c4c2851abc0e6b626/src/sentry/options/defaults.py#L23)):*
<details><summary><i>Previously, mattrobenolt (Matt Robenolt) wrote…</i></summary>
> Add `FLAG_PRIORITIZE_DISK` flag.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/4043)*

**Comment by mattrobenolt** [comment]
Reviewed 2 of 4 files at r1, 1 of 1 files at r2.
Review status: 3 of 4 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/sentry/utils/support.py, line 1 \[r2\]](https://reviewable.io:443/reviews/getsentry/sentry/4043#-KQRU8q7bnjICr-WxpmJ:-KQRU8q7bnjICr-WxpmK:bzi0fk6) ([raw file](https://github.com/getsentry/sentry/blob/e7438986eb0d179fcde9398bb761307e8541f8f4/src/sentry/utils/support.py#L1)):*
> ```Python
> def get_support_mail():
> ```

I think this completely separate utility for a single `or` is a bit overkill, but whatevz.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/4043)*

**Comment by mattrobenolt** [comment]
Reviewed 1 of 1 files at r3.
Review status: 3 of 4 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/4043)*

**Pull request merged**